### PR TITLE
Get products list

### PR DESCRIPTION
**Pull request by ThomasG51**

### CODE REVIEW

File: src/Controller/ProductController.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use App\Repository\ProductRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
```
Comment (TBoileau) [inline src/Controller/ProductController.php:9]:
```
/**
     * @Route("/api/products")
     */
```

File: src/Controller/ProductController.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class ProductController extends AbstractController
+{
+    /**
+     * Get products list
+     * @Route("/api/products", name="api_get_products", methods={"GET"})
```
Comment (TBoileau) [inline src/Controller/ProductController.php:14]: `@Route("/", name="api_get_products", methods={"GET"})`

**Review by TBoileau — CHANGES_REQUESTED** [review]

File: src/Controller/ProductController.php (hunk 1 above)
Comment (ThomasG51) [reply src/Controller/ProductController.php:10]: modification faite

File: src/Controller/ProductController.php (hunk 2 above)
Comment (ThomasG51) [reply src/Controller/ProductController.php]: modification faite

**Pull request merged**